src/main/java/com/nirmata/workflow/details/internalmodels/StartedTask.java

```diff
@@ -23,11 +23,13 @@
 {
     private final String instanceName;
     private final LocalDateTime startDateUtc;
+    private int progress;
```

Comment:
JsonSerializer.newStartedTask() and JsonSerializer.getStartedTask() need to be updated for the new field.